spec/services/invoices/refresh_draft_service_spec.rb

```diff
@@ -8,22 +8,30 @@
   describe '#call' do
     let(:status) { :draft }
     let(:invoice) do
-      create(:invoice, status:, organization: subscription.organization)
+      create(:invoice, status:, organization:, customer:,)
```

Comment:
Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call, unless each item is on its own line. (https://rubystyle.guide#no-trailing-params-comma)